rmc/utils/missense_badness.py

```diff
@@ -239,7 +248,7 @@ def prepare_amino_acid_ht(
         context_ht = amino_acids_oe_ht.annotate(**context_ht[amino_acids_oe_ht.key])
         # Checkpointing so that `amino_acids_oe` can be overwritten to the same path
         context_ht = context_ht.checkpoint(
-            f"{TEMP_PATH_WITH_FAST_DEL}/amino_acids_oe.ht"
+            f"{TEMP_PATH_WITH_FAST_DEL}/amino_acids_oe.ht", overwrite=True
         )
         # Overwrite is set to True here to ensure newly-added columns are written out
         context_ht.write(amino_acids_oe.path, overwrite=True)
```

Comment:
```suggestion
        context_ht.write(amino_acids_oe.path, overwrite=overwrite_output)
```
this overwrite can now be set to `overwrite_output`, and the comment above can be removed, right?